src/main/kotlin/com/team4099/robot2022/auto/AutonomousSelector.kt

```diff
@@ -1,40 +1,36 @@
 package com.team4099.robot2022.auto
 
-import com.team4099.lib.pathfollow.trajectoryFromPathfinder
+import com.team4099.lib.pathfollow.trajectoryFromPathPlanner
 import com.team4099.robot2022.commands.drivetrain.AutoDriveCommand
-import com.team4099.robot2022.subsystems.Drivetrain
 import edu.wpi.first.math.geometry.Rotation2d
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard
 import edu.wpi.first.wpilibj.smartdashboard.SendableChooser
-import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.CommandBase
+import edu.wpi.first.wpilibj2.command.InstantCommand
 
-class AutonomousSelector {
-  companion object {
-    private var orientationChooser: SendableChooser<Rotation2d>? = null
-    private var autonomousModeChooser: SendableChooser<AutonomousMode>? = null
+object AutonomousSelector {
+  private var orientationChooser: SendableChooser<Rotation2d> = SendableChooser()
+  private var autonomousModeChooser: SendableChooser<AutonomousMode> = SendableChooser()
 
-    init {
-      val autoTab = Shuffleboard.getTab("Auto settings")
-      orientationChooser = SendableChooser()
-      orientationChooser!!.setDefaultOption("Forward", Rotation2d.fromDegrees(0.0))
-      orientationChooser!!.addOption("Backwards", Rotation2d.fromDegrees(180.0))
-      orientationChooser!!.addOption("Left", Rotation2d.fromDegrees(90.0))
-      orientationChooser!!.addOption("Right", Rotation2d.fromDegrees(270.0))
-      autoTab.add("Starting Orientation", orientationChooser)
-      autonomousModeChooser = SendableChooser()
-      autonomousModeChooser!!.addOption("Test", AutonomousMode.TEST_AUTO_PATH)
-      autoTab.add("Mode", autonomousModeChooser)
-    }
+  init {
+    val autoTab = Shuffleboard.getTab("Auto settings")
+    orientationChooser.setDefaultOption("Forward", Rotation2d.fromDegrees(0.0))
```

Comment:
imo the `orientationChooser` isn't necessary since every auto mode should run some path which has its own orientation built in, but if we want to keep it we should use our own `Angle` stuff rather than the WPILib `Rotation2d` (so `0.degrees` etc here)